packages/mobile/ios/Podfile.lock

```diff
@@ -4,7 +4,7 @@ PODS:
   - Adjust/Core (4.26.1)
   - Analytics (4.1.3)
   - boost-for-react-native (1.63.0)
-  - CeloBlockchain (0.0.336)
+  - CeloBlockchain (0.0.351)
```

Comment:
It was updated to an intermediate one, not the latest. Not sure if this is the best PR to update but since I was already updating `react-native-geth` and I had to play around with this version a bit I decided to leave the upgrade